index.html

```diff
@@ -0,0 +1,167 @@
+                        </a></li>
+                    <li><a href="#" class="eventCards">
+                            <article>
+                                <i class="fa-solid fa-headset fa-2x"></i>
+                                <p class="eventTitle">Wizkid's Sing Along Session</p>
+                            </article>
+                            <p>Lorem ipsum dolor sit amet consectetur
+                                adipisicing elit. Officia quia asperiores dicta error</p>
+                        </a></li>
+                    <li><a href="#" class="eventCards">
+                            <article>
+                                <i class="fa-solid fa-headset fa-2x"></i>
+                                <p class="eventTitle">Essence Cover</p>
+                            </article>
+                            <p>Lorem ipsum dolor sit amet consectetur
+                                adipisicing elit. Officia quia asperiores dicta error</p>
+                        </a></li>
+                </ul>
+                <input type="button" value="Join Made in Lagos Wizkid Fest 2023" id="joinBtn">
+            </section>
+
+            <!--DYNAMIC SPEAKERS WINDOW-->
+            <section class="speakers">
+                <h4>Featured Talents</h4>
+                <ul class="speakerCards" id="speakerSection">
+
+                </ul>
+                <button class="dropdown" id="drop">MORE <span class="drop">&#8744;</span></button>
+                <div class="speakersFull" id="speakersFull">
+                    <ul class="speakerCards" id="speakerFullSection">
+                    </ul>
+                </div>
+            </section>
+        </section>
+        <section id="aboutpage">
+            <section class="welcome-section">
+                <p class="hello">"Hello Nigeria!"</p>
+                <h1>"MADE IN LAGOS" WIZKID FEST 2023</h1>
+                <p class="welcomeP aboutP">The Music Fest brings together the community <br> of experts,professionals
+                    and
+                    talented
+                    musicians home and away <br>we host the Music Fest concert in a <br> different country every two
+                    years and this year we are bringing it home!.
+                    This
+                    year, we hope to expand our invitation list including <br> organizations and individuals.</p>
+                <p class="welcomeP welcomeP2 aboutP1">Please contact us via Email for any further questions about the
+                    Made
+                    in Lagos Music Fest 2023!</p>
+                <a href="#" id="email">lanzzd191@gmail.com</a>
+            </section>
+            <section class="praises" id="praises">
+                <h3>"Made i Lagos" <br> Wizkid Fest 2023 Logo</h3>
+                <p class="welcomeP praisesP">The logo of CC Global Summit was decided through the logo competition from
+                    8. June
+                    to 7. July</p>
```

Comment:
- [ ] Kindly align your content to the middle as is in the given design as this is a requirement.

Your design
![image](https://user-images.githubusercontent.com/81932478/217587299-7de34e3b-4ec1-471e-b162-96aa2f97691d.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/217587386-4810b67f-6928-45d3-8683-447f60b57c5f.png)